mlflow/projects/__init__.py

```diff
@@ -534,6 +534,13 @@ def _get_docker_command(image, active_run, docker_args=None, volumes=None, user_
     cmd = [docker_path, "run", "--rm"]
 
     if docker_args:
+        # Allocate a pseudo-TTY if specified
+        if 't' in docker_args:
```

Comment:
Once the changes described in https://github.com/mlflow/mlflow/pull/3061/files#r451860120 have been made, I don't think we'll need this special casing logic for `t`. We can probably remove this code.